### ISSUE TITLE
The "ZipDirectory" task was not found.

### ISSUE THREAD

**Issue by StevenTCramer** [MicrosoftDocs/visualstudio-docs#1244]
.net core 2.1 console app and added the example above and I get 

```
The "ZipDirectory" task was not found. Check the following: 1.) The name of the task in the project file is the same as the name of the task class. 2.) The task class is "public" and implements the Microsoft.Build.Framework.ITask interface. 3.) The task is correctly declared with &lt;UsingTask&gt; in the project file, or in the *.tasks files located in the "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin" directory. 
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4c2cbd60-5d56-af4f-b196-4c9f190a1586
* Version Independent ID: 6408870a-85cd-428c-5ede-2338804481d6
* Content: [ZipDirectory Task - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/msbuild/zipdirectory-task)
* Content Source: [docs/msbuild/zipdirectory-task.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/msbuild/zipdirectory-task.md)
* Product: **visual-studio-dev15**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

**Comment by StevenTCramer** [comment]
It is because 15.7 is the current released version and this is for 15.8.

Sorry!

**Comment by Mikejo5000** [comment]
No problem. Thanks for the follow up!
